base/exports.jl

```diff
@@ -1063,3 +1063,71 @@ export
     @view,
     @views,
     @static
+
+# TODO: use normal syntax once JuliaSyntax.jl becomes available at this point in bootstrapping
+eval(Expr(:public,
+    Symbol("@assume_effects"),
+    Symbol("@constprop"),
+    Symbol("@locals"),
+    Symbol("@propagate_inbounds"),
+    :AbstractLock,
+    :AsyncCondition,
+    :CFunction,
+    :Checked,
+    :CodeUnits,
+    :DL_LOAD_PATH,
+    :EnvDict,
+    :Event,
+    :Fix1,
+    :Fix2,
+    :Generator,
+    :ImmutableDict,
+    :IteratorEltype,
+    :IteratorSize,
+    :OneTo,
+    :Pairs,
+    :Semaphore,
+    :UUID,
+    :acquire,
+    :active_project,
+    :add_sum,
+    :cconvert,
+    :checked_length,
+    :compilecache,
```

Comment:
`compilecache` is internal
and the others are probably implementation defined so people shouldn't rely on it?

Reply:
```suggestion
```